org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WListbox.java

```diff
@@ -789,20 +798,21 @@ public Integer getSelectedRowKey()
 	 * @param index
 	 * @return
 	 */
-	public Integer getRowKeyAt (int index){
+	public Object getRowKeyAt (int index){
```

Comment:
this and getSelectedRowKey above is a breaking change, needs to add to migration note.

Reply:
I've not test but I think `public <T extends Object> getSelectedRowKey()` and cast return type with `(T)` probably make it more backward compatible.

Reply:
Or `<T extends Serializable>`

Reply:
Thanks @hengsin - the suggested change worked fine - the Integer casting is not needed after this.